http.md

```diff
@@ -0,0 +1,291 @@
+    "params": [
+        "4hXTCkRzt9WyecNzV1XPgCDfGAZzQKNxLXgynz5QDuWWPSAZBZSHptvWRL3BjCvzUXRdKvHL2b7yGrRQcWyaqsaBCncVG7BFggS8w9snUts67BSh3EqKpXLUm5UMHfD7ZBe9GhARjbNQMLJ1QD3Spr6oMTBU6EhdB4RD8CP2xUxr2u3d6fos36PD98XS6oX8TQjLpsMwncs5DAMiD4nNnR8NBfyghGCWvCVifVwvA8B8TJxE1aiyiv2L429BCWfyzAme5sZW8rDb14NeCQHhZbtNqfXhcp2tAnaAT",
+        "5hXTCkRzt9WyecNzV1XPgCDfGAZzQKNxLXgynz5QDuWWPSAZBZSHptvWRL3BjCvzUXRdKvHL2b7yGrRQcWyaqsaBCncVG7BFggS8w9snUts67BSh3EqKpXLUm5UMHfD7ZBe9GhARjbNQMLJ1QD3Spr6oMTBU6EhdB4RD8CP2xUxr2u3d6fos36PD98XS6oX8TQjLpsMwncs5DAMiD4nNnR8NBfyghGCWvCVifVwvA8B8TJxE1aiyiv2L429BCWfyzAme5sZW8rDb14NeCQHhZbtNqfXhcp2tAnaAT",
+        "6hXTCkRzt9WyecNzV1XPgCDfGAZzQKNxLXgynz5QDuWWPSAZBZSHptvWRL3BjCvzUXRdKvHL2b7yGrRQcWyaqsaBCncVG7BFggS8w9snUts67BSh3EqKpXLUm5UMHfD7ZBe9GhARjbNQMLJ1QD3Spr6oMTBU6EhdB4RD8CP2xUxr2u3d6fos36PD98XS6oX8TQjLpsMwncs5DAMiD4nNnR8NBfyghGCWvCVifVwvA8B8TJxE1aiyiv2L429BCWfyzAme5sZW8rDb14NeCQHhZbtNqfXhcp2tAnaAT",
+        "7hXTCkRzt9WyecNzV1XPgCDfGAZzQKNxLXgynz5QDuWWPSAZBZSHptvWRL3BjCvzUXRdKvHL2b7yGrRQcWyaqsaBCncVG7BFggS8w9snUts67BSh3EqKpXLUm5UMHfD7ZBe9GhARjbNQMLJ1QD3Spr6oMTBU6EhdB4RD8CP2xUxr2u3d6fos36PD98XS6oX8TQjLpsMwncs5DAMiD4nNnR8NBfyghGCWvCVifVwvA8B8TJxE1aiyiv2L429BCWfyzAme5sZW8rDb14NeCQHhZbtNqfXhcp2tAnaAT",
+        {
+            "commitment": "finalized"
+        }
+    ]
+}
+'
+```
+
+#### Response
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": "2id3YC2jK9G5Wo2phDx4gJVAew8DcY5NAojnVuao8rkxwPYPe8cSwE5GzhEgJA2y8fVjDEo6iR6ykBvDxrTQrtpb",
+    "id": 1
+}
+```
+
+## getBundle
+
+Returns bundle details for a submitted bundle
+
+### Parameters
+
+- `bundle_id`: `string`  a base-58/64 encoded string (required).
+- `object` (optional)
+  - Configuration object containing the following fields (optional):
+    - `commitment`: String. [Configure state commitment](#configuring-state-commitment). Default: `finalized`.
+    - `encoding`: String (Default: "json"). Encoding for the returned Transaction. Possible values: json, base64, base58.
+
+### Result
+
+- `<null>`: If the bundle is not found.
+- `<object>`: If the bundle is found, an object with the following fields:
+  - `bundle_id`: `string`  a base-58/64 encoded string (required).
+  - `slot_id`: `<u64>`  The slot this bundle was processed in.
+  - `meta`: `<object|null>`: Transaction status metadata object:
+    - `err`: `<object|null>`: Error if the transaction failed, null if the transaction succeeded/finalized.
+
+### Code sample
+
+### Request
+
+```bash
+curl http://localhost:8899 -X POST -H "Content-Type: application/json" -d '
+{
+    "jsonrpc": "2.0",
+    "id": 1,
+    "method": "getBundle",
+    "params": [
+        "2nBhEBYYvfaAe16UMNqRHre4YNSskvuYgx3M6E4JP1oDYvZEJHvoPzyUidNgNX5r9sTyN1J9UxtbCXy2rqYcuyuv",
+      {
+            "commitment": "finalized"
+      }
+    ]
+}
+'
+```
+
+### Response
+
+```json
+{
+    "jsonrpc": "2.0",
+    "result": {
+        "bundle_id": "3Eq21vXNB5s86c62bVuUfTeaMif1N2kUqRPBmGRJhyTA",
+        "slot": 1234,
+        "meta": {
```

Comment:
What other bundle details need to be exposed? Or are the states below sufficient?